package.json

```diff
@@ -2,17 +2,9 @@
   "name": "technical_metadata",
```

Comment:
Do we still need package.json now that most of its contents have been removed?

Reply:
I don't think so, but it's possible someone will need something in the future. How about we follow up after to pursue this.